gramineproject/gramine
# [pkg] Initial `debian/`

## Description of the changes 

Packaging files for debian

## How to test this PR? 

Install dependencies, then `debuild`.
New job in CI should pass.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/gramineproject/gramine/512)
